### PR TITLE
SOLR-13289: Add Refguide changes

### PR DESCRIPTION
**Pull request by tflobbe**
Reference guide changes for SOLR-13289

### CODE REVIEW

File: solr/solr-ref-guide/src/common-query-parameters.adoc [hunk 1]
```diff
@@ -361,3 +361,42 @@ This is what happens if a similar request is sent that adds `echoParams=all` to
...
+
+[source,text]
+q=quick brown fox&minExactHits=100&rows=10
+
+[source,json]
+----
+"response": {
+    "numFound": 153,
+    "start": 0,
+    "hitCountExact": false,
```
Comment (dsmiley) [inline solr/solr-ref-guide/src/common-query-parameters.adoc:384]: didn't we agree on "numFoundExact" ?

Comment (tflobbe) [reply solr/solr-ref-guide/src/common-query-parameters.adoc:384]: Ah, right, I copied from the Jira issue and forgot to update

File: solr/solr-ref-guide/src/common-query-parameters.adoc [hunk 2]
```diff
@@ -361,3 +361,42 @@ This is what happens if a similar request is sent that adds `echoParams=all` to
   }
 }
 ----
+
+== minExactHits Parameter
```
[the remark this answers is not in the document]
Comment (tflobbe) [reply solr/solr-ref-guide/src/common-query-parameters.adoc:365]: Yes, I think that makes sense

[the remark this answers is not in the document]
Comment (tflobbe) [reply solr/solr-ref-guide/src/common-query-parameters.adoc]: Discussed in https://issues.apache.org/jira/browse/SOLR-13289

**Pull request merged**